module_utils/network_lsr/argument_validator.py

```diff
@@ -1182,17 +1183,17 @@ def __init__(self):
         )
 
     def _validate_post(self, value, name, result):
-        if result["key_mgmt"] == "wpa-psk":
+        if result["key_mgmt"] == "wpa-psk" or result["key_mgmt"] == "owe":
             if result["password"] is None:
                 raise ValidationError(
                     name,
-                    "must supply a password if using 'wpa-psk' key management",
+                    "must supply a password if using 'wpa-psk' or 'owe' key management",
                 )
         else:
             if result["password"] is not None:
                 raise ValidationError(
                     name,
-                    "password only allowed if using 'wpa-psk' key management",
+                    "password only allowed if using 'wpa-psk' or 'owe' key management",
```

Comment:
The same as above.

Reply:
can not be the same as above in the sense that error was raised when:
 -  `result["key_mgmt"] == 'owe'`
 - `result["password"] is not None `

Reply:
my fault!